akka-docs/rst/java/code/docs/pattern/SchedulerPatternTest.java

```diff
@@ -148,20 +149,20 @@ public void testSchedule(final JavaTestKit probe, Props props,
         system.eventStream().publish(new Mute(filter));
 
         final ActorRef actor = system.actorOf(props);
-        new Within(startDuration) {
-          protected void run() {
-            probe.expectMsgEquals("tick");
-            probe.expectMsgEquals("tick");
-            probe.expectMsgEquals("tick");
-          }
-        };
+        within(startDuration, () -> {
+          probe.expectMsgEquals("tick");
+          probe.expectMsgEquals("tick");
+          probe.expectMsgEquals("tick");
+          return null;
```

Comment:
Since `within` is delegated to scaladsl which needs type T, so I have to put an ugly `return null` here.